src/client/client.go

```diff
@@ -1399,6 +1399,7 @@ func (s *SignalClient) UpdateProfile(number string, profileName string, base64Av
 			Name         string `json:"given-name"`
 			Avatar       string `json:"avatar,omitempty"`
 			RemoveAvatar bool   `json:"remove-avatar"`
+			About        string `json:"about"`
```

Comment:
Same here. I think a pointer would also be better here. Please also provide the `omitempty` (see https://github.com/bbernhard/signal-cli-rest-api/blob/master/src/client/client.go#L1613 for an example)

Reply:
See 57dcb2f